apps/website/src/pages/docs/avatar.mdx

```diff
@@ -99,12 +99,18 @@ You can also have the stacked group animate to unstacked on hover. When doing th
 
 ### Overflow
 
-A maximum of six avatars can be shown within a group. Additional avatars are hidden; however, the number of hidden items should be displayed in the last avatar circle as shown below. Additionally, you could display a [tooltip](tooltip) on hover, a [dialog](dialog) on click, or perhaps a [dropdown menu](dropdownmenu) of the hidden users. Implementation of these options depends on the purpose of the page and the use case.
+By default a maximum of five avatars will be shown within a group. Additional avatars are hidden; however, the number of hidden items will be displayed in the last avatar circle as shown below.
```

Comment:
> By default a maximum of five avatars will be shown within a group

I did not test if this is true or not.  Do we actually limit them to 5?  If we don't, it could be rephrased to:

> It is strongly suggested you display a maximum of five avatars within a group

Reply:
We don't limit them to five. They can choose however many they want, but it defaults to 5

Reply:
Ah I understand, the way it is worded seems to be correct then.  LGTM.

Reply:
Updated.

Reply:
Oh, I didn't see your response when I pushed my code. I'll undo it then.